components/Form/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,155 @@
+  value?: string | null;
+  inputClassName?: string;
+}
+
+interface Props extends IAutocomplete {
+  control: Control;
+  onChange: (arg0: string | null) => void;
+  value?: string;
+}
+
+// eslint-disable-next-line react/display-name
+export const Autocomplete = forwardRef<HTMLInputElement, AutoProps>(
+  (
+    {
+      label,
+      options,
+      hint,
+      name,
+      error,
+      labelSize = 'm',
+      required,
+      onChange,
+      placeholder,
+      width = 20,
+      govGrid,
+      value,
+      inputClassName,
+      ...otherProps
+    }: AutoProps,
+    ref
+  ) => {
+    const optionObject = options?.map((option: OptionObject) => {
+      if (typeof option === 'string') {
+        return { value: option, text: option };
+      }
+      return option;
+    });
+    const items = sortObject(optionObject);
+
+    const getTextValue = () => {
+      if (value) {
+        const selection = items.find(
+          (item: OptionObject) => item.value === value
+        );
+        return selection.text;
+      }
+      return '';
+    };
+
+    return (
+      <Downshift
+        initialInputValue={getTextValue()}
+        onChange={(selection) => onChange(selection ? selection.value : '')}
+        itemToString={(item) => (item ? item.text : '')}
+      >
+        {({
+          getInputProps,
+          getItemProps,
+          getMenuProps,
+          isOpen,
+          inputValue,
+          toggleMenu,
+          highlightedIndex,
+          selectedItem,
+          clearSelection,
+        }) => {
+          return (
+            <div
+              className={cx(`govuk-form-group govuk-grid-column-${govGrid}`, {
+                'govuk-form-group--error': error,
+              })}
+            >
+              <label
+                className={`govuk-label govuk-label--${labelSize}`}
+                htmlFor={name}
+              >
+                {label} {required && <span className="govuk-required">*</span>}
+              </label>
+              <AutoInput
+                ref={ref}
+                name={name}
+                items={items}
+                value={value}
+                error={error}
+                label={label}
+                hint={hint}
+                placeholder={placeholder}
+                width={width}
+                inputClassName={inputClassName}
+                getInputProps={getInputProps}
+                getItemProps={getItemProps}
+                getMenuProps={getMenuProps}
+                isOpen={isOpen}
+                inputValue={inputValue}
+                toggleMenu={toggleMenu}
+                clearSelection={clearSelection}
+                highlightedIndex={highlightedIndex}
+                selectedItem={selectedItem}
+                {...otherProps}
+              />
+            </div>
+          );
+        }}
+      </Downshift>
+    );
+  }
+);
+
+const ControlledAutocomplete = ({
+  name,
+  control,
+  onChange,
+  value,
```

Comment:
These `onChange` and `value` are not being used as they are redefined in the `render` of the `Controller`